runtime/vm/JFRChunkWriter.hpp

```diff
@@ -193,6 +194,38 @@ class VM_JFRChunkWriter {
 		return false;
 	}
 
+	U_8 *reserveCheckpointSize() {
+		if (J9JAVAVM_COMPRESS_OBJECT_REFERENCES(_vm)) {
+			return _bufferWriter->getAndIncCursor(sizeof(U_32));
+		} else {
+			return _bufferWriter->getAndIncCursor(sizeof(U_64) + sizeof(U_8));
```

Comment:
The increments should be 4 or 9 (to match the space consumed when the value is actually written).